features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/base64/Base64Serializer.kt

```diff
@@ -133,7 +160,9 @@ class Base64Serializer private constructor(
             base64LRUCache?.put(drawable, base64Result)
         }
 
-        bitmapPool?.put(bitmap)
+        if (enrichedBitmap.shouldCache) {
+            bitmapPool?.put(enrichedBitmap.bitmap)
```

Comment:
I have a feeling that we don't really need `EnrichedBitmap` maybe? Like we can have just 2 arguments of the methods - bitmap and caching flag. Anyway this class seems to be used only inside `Base64Serializer`. wdyt?